index.src.html

```diff
@@ -3835,6 +3835,10 @@
 : <dfn dict-member for=PlaneLayout>offset</dfn>
 :: The offset in bytes where the given plane begins within a {{BufferSource}}.
 
+    NOTE: Authors can use {{PlaneLayout/offset}} to create a gap of bytes
```

Comment:
The bigger concern is with the gaps between `row_bytes` and `stride`. I'd actually lean towards a guarantee that bytes outside of `stride * rows` between planes are in fact unchanged by `copyTo()`.